openstf/pipeline/train_predict_backtest.py

```diff
@@ -0,0 +1,135 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+from typing import Union, List
+from pathlib import Path
+import logging
+
+import pandas as pd
+
+from openstf.model.confidence_interval_applicator import ConfidenceIntervalApplicator
+from openstf.postprocessing.postprocessing import (
+    add_prediction_job_properties_to_forecast,
+)
+from openstf.model.standard_deviation_generator import StandardDeviationGenerator
+from openstf.model.model_creator import ModelCreator
+from openstf.metrics.reporter import Reporter
+from openstf.model.serializer import PersistentStorageSerializer
+from openstf.model_selection.model_selection import split_data_train_validation_test
+from openstf.validation import validation
+from openstf.feature_engineering.feature_applicator import TrainFeatureApplicator
+
+DEFAULT_TRAIN_HORIZONS: List[float] = [0.25, 24.0]
+DEFAULT_EARLY_STOPPING_ROUNDS: int = 10
+
+
+def train_model_and_forecast_back_test(
```

Comment:
I have not looked at the implementation yet. But if this pipeline is a combination of the `create_forecast` and `train_model` pipeline for the backtest usecase I would suggest to call it `train_model_and_create_forecast_backtest` (one wordt backtest is also consistent with the module name)